sampleconfig/configtx.yaml

```diff
@@ -500,6 +504,7 @@ Profiles:
         <<: *ChannelDefaults
         Orderer:
             <<: *OrdererDefaults
+            OrdererType: solo
             Organizations:
                 - <<: *SampleOrg
```

Comment:
I don’t want to balloon the size of this change so I think this is a separate issue, but, the `SampleOrgPolicies` reference has the same sort of issue where `Admins` is defined in the anchor and then defined again after the reference.

@jyellick - Do you have any thoughts here? It looks like the structure of the yaml is relying quite a bit on the implementation of the yaml parser in a way that is not strictly spec compliant. I’m not sure how far we want to go down the path of making this “correct”.